routes/usersRouter.js

```diff
@@ -9,41 +9,80 @@ const {
   getUserByUsername,
   getUser,
   getAllUsers,
+  getUserById,
 } = require('../db/users');
 
 const {
 	createUser,
 } = require('../db/index');
 
 
+usersRouter.get('/', async (req, res, next) => {
```

Comment:
REQUIRED - Should be only accessible by admin users.

Reply:
requireAdmin applied.